src/java_bytecode/java_bytecode_language.h

```diff
@@ -34,8 +37,10 @@ class java_bytecode_languaget:public languaget
 
   void show_parse(std::ostream &out) override;
 
-  ~java_bytecode_languaget() override;
-  java_bytecode_languaget() { }
+  virtual ~java_bytecode_languaget();
+  java_bytecode_languaget():
+    max_nondet_array_length(5),
```

Comment:
this is the default value for the user code arrays, is having a `#define` better?